txp/Pos/Txp/Logic/Local.hs

```diff
@@ -62,31 +67,39 @@ txProcessTransaction itw@(txId, txAux) = do
                    toList $
                    NE.zipWith (liftM2 (,) . Just) _txInputs resolvedOuts
     pRes <- modifyTxpLocalData $
-            processTxDo resolved toilEnv tipDB itw
+            processTxDo resolved toilEnv tipDB itw bootEra
     case pRes of
         Left er -> do
             logDebug $ sformat ("Transaction processing failed: "%build) txId
             throwError er
         Right _   ->
             logDebug (sformat ("Transaction is processed successfully: "%build) txId)
   where
+    notBootRelated =
+        let txDistr = getTxDistribution $ taDistribution txAux
+            inBoot (s,_) = s `HS.member` genesisBootProdStakeholders
```

Comment:
It is fundamentally wrong, we can't hardcode list of genesis stakeholders. However, as I said, I don't mind if it's not fixed in this issue. I will fix it by myself after this one is merged.